### ISSUE TITLE
Secret Store nodes use incorrect gas settings for making contract transactions so they never get through

### ISSUE THREAD

**Issue by ngyam** [paritytech/parity#11773]
- **OpenEthereum version**: v2.6.7
- **Operating system**: Linux Ubuntu 16 LTS
- **Installation**:  built from source
- **Fully synchronized**: yes
- **Network**: Volta
- **Restarted**: yes

Problem is explained in this Secret Store repo issue, but I thought it's better to also raise it here: [https://github.com/paritytech/secret-store/issues/52](https://github.com/paritytech/secret-store/issues/52)

**TL;DR**: would it be possible to add 2 config params for the secret store module to set used gasprice and gas for transactions, or calculate them more sensibly?
